.github/workflows/build-and-deploy.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        container: [ "python:3.8", "python:3.9", "python:3.10" ]
+        container: [ "python:3.10" ]
```

Comment:
Because we are deploying the package we only need to build in version of python. We only need to run the different versions of python when we are running the tests to make sure they are compatible